### PR TITLE
updating site text per NBs comments

### PR DESCRIPTION
**Pull request by eread-usgs**
@dblodgett-usgs or @lindsaycarr can you review.

### CODE REVIEW

File: data/siteText.yaml [hunk 1]
```diff
@@ -1,27 +1,29 @@
...
   One million gallons of water covers a football field in about 2.8 feet of water. 
   
-  Every 5 years since 1950, the USGS has compiled and estimated water-use information in cooperation with State, 
-  Federal, and local agencies to document how the Nation's water resources are used. Prior to 1960, Industrial and Thermoelectric 
-  water use categories were not assessed individually. Data at the site, county, 
-  and major watershed levels are available for some years, but not for the full period of record. Compilation of water-use data for 2015 is underway. 
-  This information is essential to accurately understand how future water demands will be met while maintaining 
-  adequate water quality and quantities for human and ecosystem needs. The USGS Water Availability and Use Program hopes to move toward even higher spatial resolution 
-  and more frequent reports in the future.
+  The categories used for assessing water use have varied through time, as shown [here](http://water.usgs.gov/watuse/WU-Category-Changes.html).
```
Comment (jiwalker-usgs) [inline data/siteText.yaml:24]: nice use of the markdown :+1:

File: data/siteText.yaml [hunk 2]
```diff
@@ -1,27 +1,29 @@
...
   and the bars for water use values in million gallons of water per day (Mgal/d).
   One million gallons of water covers a football field in about 2.8 feet of water. 
   
-  Every 5 years since 1950, the USGS has compiled and estimated water-use information in cooperation with State, 
-  Federal, and local agencies to document how the Nation's water resources are used. Prior to 1960, Industrial and Thermoelectric 
-  water use categories were not assessed individually. Data at the site, county, 
-  and major watershed levels are available for some years, but not for the full period of record. Compilation of water-use data for 2015 is underway. 
-  This information is essential to accurately understand how future water demands will be met while maintaining 
-  adequate water quality and quantities for human and ecosystem needs. The USGS Water Availability and Use Program hopes to move toward even higher spatial resolution 
-  and more frequent reports in the future.
```
Comment (lindsayplatt) [inline data/siteText.yaml:24]: Seems like by deleting all of this info we are losing the main grab/take home point of the viz - "data collected sparsely, but it's important so support it". I would rather see some more information about the why than so many details about the categories - that doesn't really interest the public which is why we refrained from getting into it (although I see how we should explain why there's not data in some of the years).

**Comment by eread-usgs** [comment]
check it out @lindsaycarr

**Comment by eread-usgs** [comment]
ok, up for merging @lindsaycarr ? @jiwalker-usgs ? anyone?

**Pull request merged**